core/src/main/scala/pureconfig/configurable/package.scala

```diff
@@ -0,0 +1,20 @@
+package pureconfig
+
+import scala.util.Try
+import java.time.{ LocalDate, LocalDateTime, LocalTime }
+import java.time.format.DateTimeFormatter
+
+package object configurable {
```

Comment:
I aspire to add a `pureconfig-joda` JAR once this gets merged. 
@melrief Would you expect that the similar methods in that JAR would go in a `pureconfig.joda.configurable` package or would you have them load them into this namespace?
I ask more from a philosophical standpoint of how we want to structure the library's API. I don't really have a good answer to that question.